### ISSUE TITLE
v2.7: Missing messages

### ISSUE THREAD

**Issue by jgarciaroca** [crs4/hl7apy#131]
List of missing messages: ADT_A04, ADT_A07, ADT_A08, ADT_A10, ADT_A11, ADT_A13, ADT_A14, ADT_A22, ADT_A23, ADT_A25, ADT_A26, ADT_A27, ADT_A28, ADT_A29, ADT_A31, ADT_A32, ADT_A33, ADT_A40, ADT_A41, ADT_A42, ADT_A47, ADT_A49, ADT_A51, ADT_A53, ADT_A55, MFN_M14, PMU_B02, PMU_B05, PMU_B06

More might be missing, i just detected those doing a sequential search of existing message gaps.

**Comment by svituz** [comment]
Hi @jgarciaroca, I read the documentation, and all the ADT messages you pointed out are not actually valid.
The A04 event must be used with the trigger event ADT_A01 and the A08 event as well. Similarly, the A07 event should be used with the ADT_A06 trigger event, and so on. Same thing for PMU trigger events (B05 should be used with PMU_B01, B05 and B06 with PMU_B04).
Regarding the M14 event for MFN, that should be used with Znn messages which I admit I wasn't aware of. The support for this kind of Z trigger events should be handled in another way probably.

**Comment by jgarciaroca** [comment]
You are right. When i created the PR i didn't realized that the test messages i was trying to parse where being send wrong. the MSH.9 received was pointing to "ADT^A04^ADT_A04" instead of "ADT^A04^ADT_A01", so the parser couldn't parse that message.
